build.rs

```diff
@@ -0,0 +1,56 @@
+//! Reads the `templates/` directory and includes all versions' templates as
+//! part of the binary to reduce installation footprint
+extern crate ron;
```

Comment:
`extern crate` is not required in Rust 2018.

Reply:
If I remove this, the build throws the following errors:
```
error[E0433]: failed to resolve: maybe a missing `extern crate ron;`?
 --> build.rs:8:5
  |
8 | use ron::de::from_reader;
  |     ^^^ maybe a missing `extern crate ron;`?

error[E0425]: cannot find function `from_reader` in this scope
  --> build.rs:18:5
   |
18 |     from_reader(File::open(&path).expect("Failed to open index.ron"))
   |     ^^^^^^^^^^^ not found in this scope
```

If I add `edition = "2018"` to `Cargo.toml`, the build throws a ton more errors

Reply:
Oh I see, we haven't upgraded yet! Never mind then.

Reply:
I can open an issue to do a general upgrade. Looks like the build's been failing due to the old liquid package. I can encompass this along with an other packages that might break the build.